lib/puppet/pops/types/execution_result.rb

```diff
@@ -36,6 +36,14 @@ def self._pcore_type
       @type
     end
 
+    # Creates a pure Data hash from a result hash returned from the Bolt::Executor
+    # @return [Hash{String => Data}] The data hash
```

Comment:
is this really the yard way of writing that? Thought it was something like Hash<String, Data> ??